acquia_lift.module

```diff
@@ -2853,35 +2778,16 @@ function acquia_lift_command_messagebox($message, $seconds = 0) {
 }
 
 /**
- * Returns an AJAX command to toggle the page variation element selector.
+ * Returns an AJAX command to toggle the element variation element selector.
  *
  * @param bool $start
- *   True to start the page variation mode, FALSE to stop it.
+ *   True to start the element variation mode, FALSE to stop it.
  */
-function acquia_lift_command_page_variation_toggle($start = TRUE) {
+function acquia_lift_command_element_variation_toggle($start = TRUE) {
   return array(
     'command' => 'acquia_lift_variation_toggle',
     'data' => array(
       'start' => $start,
-      'type' => 'page',
-    ),
-  );
-}
-
-/**
- * Returns an AJAX command to preview a specific page variation.
- *
- * @param string $agent_name
- *   The name of the campaign/agent to which the variation belongs.
- * @param int $variation_index
- *   The index of the variation to preview.
- */
-function acquia_lift_command_page_variation_preview($agent_name, $variation_index) {
```

Comment:
Fixed. The reason for this error is that I accidentally removed this router configuration, and the config was brought back in 5582632a416998ab394f41da6afdeee728a87200. All works now.

This also explains why I didn't get the error until I apply my update hook - by hook execution, the menu_router table's content was regenerated, therefore taking out the record that's routing _admin/structure/acquia_lift/variation/editText_, therefore the link was loaded by default action instead of the json builder.

Lisa, to fix your local site, the hook needed to be redone with the fixed code.